### PR TITLE
Pokemon API web app

### PR DESCRIPTION
**Pull request by mikemtzp**
**IMPORTANT: when giving the review please take into consideration I was recently unpaired from my coding partner, though the project is complete, I was originally assigned to the Student B tasks. Those tasks follow the proper branch addition from different feature branches to the dev branch. At the end of this PR, a screen capture that avails the impairment is added.**

Developed a web app that displays the first-generation Pokémon:

- Get a list of items with a unique item id (or generate the unique id).
- For a given item, get detailed information about it.
- Use the Involvement API to record the different user interactions (likes, and comments).
- Build these the home page and the comments popup interfaces.
- Add design including colors, typographies, spacings, etc.
- When the page loads, the web app retrieves data from:
The selected API shows the list of items on the screen.
The Involvement API to show the item likes.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
When the user clicks on the "Comments" button, the Comments popup appears.
- When the popup loads, the web app retrieves data from:
The selected API and shows details about the selected item.
The Involvement API to show the item comments.
When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
- Add counters that show the number of items (home). and the number of comments (comments popup).
- Counter functions are covered with unit tests using Jest.
- Set up the repository on GitHub and use Gitflow.
- Set up webpack.
- Set up a JavaScript testing library (Jest).

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+Give a ⭐️ if you like this project!
+
+## 🥇 Acknowledgments
+
+- This README.md was elaborated with the [Microverse readme-template](https://github.com/microverseinc/readme-template)
+
+## 📝 License
+
+[MIT License](https://github.com/mikemtzp/PokemonAPI/blob/master/MIT.md)
```
Comment (ahmetbozaci) [inline README.md:63]: - [ ]  To make a professional readme file kindly fix the broken link.

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+## 🛠️ Built With
+
+- HTML, CSS & Javascript
+- Webpack
+- Jest
+- Babel
+- Linters: Lighthouse, Webhint, Stylelint & ESlint
+
+## 📹 [Video documentation](https://www.loom.com/share/febaae743e004d9aa35818b252fdca95)
+
```
Comment (ahmetbozaci) [inline README.md:13]: - [ ] Kindly add a live link to the website.

File: src/modules/counter-comments.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-expressions */
```
Comment (ahmetbozaci) [inline src/modules/counter-comments.js:1]: - [ ] Please don't disable eslint. Kİndly open it and fix the error.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @mikemtzp  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Presentation video is very professional. You show the demo and some codes.✔️
- You know how to use ES6 and Webpack.✔️
- Well done with following JavaScript best practices. The codes look so clean.
- You added a jest test for counter function and all tests passed.

___


### ♻️ Request changes ♻️

- [x] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: README.md (hunk 2 above)
Comment (mikemtzp) [reply README.md:13]: Hello, @ahmetbozaci Thanks for the review 😁 

Right now when you click on the video documentation link it opens the loom video
Could you explain this point in more detail? What else should I add to it?

File: README.md (hunk 1 above)
Comment (mikemtzp) [reply README.md]: @ahmetbozaci Link fixed, thanks for noting it.😁

File: README.md (hunk 2 above)
Comment (ahmetbozaci) [reply README.md:13]: - Yes kindly deploy and add the live link.Video link is working.

Comment (mikemtzp) [reply README.md]: @ahmetbozaci Thanks for clarifying it. Live link added. 👌🏼

File: src/modules/counter-comments.js (hunk 3 above)
Comment (mikemtzp) [reply src/modules/counter-comments.js]: @ahmetbozaci eslint disable removed and fixed. 👍🏼

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@mikemtzp  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Presentation video is very professional. You show the demo and some codes.✔️
- You know how to use ES6 and Webpack.✔️
- Well done with following JavaScript best practices. The codes look so clean.
- You added a jest test for counter function and all tests passed.


___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

aproved
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**